pkg/backuphelpers/backupvars.go

```diff
@@ -0,0 +1,66 @@
+package backuphelpers
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"sync"
+
+	backupv1alpha1 "github.com/openshift/api/config/v1alpha1"
+)
+
+var (
+	validBackupVars = map[string]struct{}{
+		"schedule":   {},
+		"timezone":   {},
+		"data-dir":   {},
+		"config-dir": {},
+		"backup-dir": {},
+	}
+)
+
+type BackupVar interface {
+	GetBackupVars() *BackupConfig
+}
+
+type BackupConfig struct {
+	enabled  bool
+	backupCR backupv1alpha1.EtcdBackupSpec
+	mux      sync.Mutex
+}
+
```

Comment:
```suggestion
func NewDisabledBackupConfig() *BackupConfig {
	return &BackupConfig{
		enabled:  false,
		mux:      sync.Mutex{},
	}
}
```

let's be clear that this is not enabled

Reply:
done